lwconfig/_examples/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lacework/go-sdk/lwconfig"
+)
+
+func main() {
+	cPath, err := lwconfig.DefaultConfigPath()
+	if err != nil {
+		fmt.Println("Unable to detect default config path location: %s", err)
```

Comment:
nit as this is an example. Should be printf
```suggestion
		fmt.Printf("Unable to detect default config path location: %s", err)
```

Reply:
```suggestion
		fmt.Printf("Unable to detect default config path location: %s\n", err)
```